Nomic Logging Error

star-nox opened the pull request:
Getting some string concatenation error in logging

KastanDay commented:
Example of the error this PR should fix: https://ncsa-zb.sentry.io/issues/4732719525/?query=is%3Aunresolved&referrer=issue-stream&stream_index=0
![CleanShot 2023-12-17 at 10 51 52](https://github.com/UIUC-Chatbot/ai-ta-backend/assets/13607221/270e4888-1542-48ce-a142-698dd12f9c95)

star-nox commented:
> Example of the error this PR should fix: https://ncsa-zb.sentry.io/issues/4732719525/?query=is%3Aunresolved&referrer=issue-stream&stream_index=0 ![CleanShot 2023-12-17 at 10 51 52](https://private-user-images.githubusercontent.com/13607221/291097098-270e4888-1542-48ce-a142-698dd12f9c95.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI5NDkwOTUsIm5iZiI6MTcwMjk0ODc5NSwicGF0aCI6Ii8xMzYwNzIyMS8yOTEwOTcwOTgtMjcwZTQ4ODgtMTU0Mi00OGNlLWExNDItNjk4ZGQxMmY5Yzk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE5VDAxMTk1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZjMWMxYWM4ZTY5ZTczYjEwMDk2M2U4MmViMjE5MGZkY2IxMmRjMzhhZTIyMmEwNDQxNWEwNTZmNmJmMGZjNDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lPMUVu4UoBJTqsZxUkXEZIqnE-2s_1SnnWJxomU4Ms0)

The `content` parameter seems to be changed. It was a string earlier, now its a list. I have added conditions to check for both types in the code. I have modified the Exception logging in \get-nomic-map.

KastanDay commented:
@star-nox You should really check on this error too:
```
ai_ta_backend/nomic_logging.py:44:1
 235:10  high  Undefined name `message`  ruff/F821           
 246:12  high  Undefined name `m`        ruff/F821        
 ```

KastanDay reviewed (approved):
Not tested, but looking good. Sentry.io will determine if this works better! :)
